ci/Jenkinsfile.android

```diff
@@ -72,9 +72,9 @@ pipeline {
         stage('Upload') {
           steps { script {
             def urls = apks.collect { s5cmd.upload(it) }
-            if (urls.size() > 1) { /* Return only the universal APK. */
-              env.PKG_URL = urls.find { it.contains('universal') }
-            } else { /* If no universal is available pick first. */
+            if (urls.size() > 1) { /* Dont Return the universal APK. */
+              env.PKG_URL = urls.find { !it.contains('universal') }
```

Comment:
Why? This doesn't make sense, might as well just have the `urls.first()` and drop the `find` since it does nothing.

Reply:
ah because I did not want to publish the universal `APK` in github since `ANDROID_ABI_SPLIT=true` setting generates both the split `APK` along with the universal `APK` :D

Reply:
Oh, right, I forgot the universal is included. We could just disable that tho right? Why not disable it in Gradle config?

Reply:
makes sense. I'll do that.